src/components/Organization/Dashboard/PricingCard.tsx

```diff
@@ -16,56 +15,52 @@ const PricingCard = ({
   width?: string
   popular?: boolean
   features: string[]
-}) => {
-  const { textColorBrand } = useDarkMode()
-
-  return (
-    <Card variant='pricing-card' width={width}>
-      <CardHeader>
-        <Text>{title}</Text>
-        <Text>{subtitle}</Text>
-      </CardHeader>
-      <CardBody>
-        <Button variant='pricing-card'>
-          <Trans i18nKey='subscribe'>Subscribe</Trans>
-        </Button>
-        <Text color='black'>
-          <Trans i18nKey='pricing_card.from'>From ${price}/year</Trans>
-        </Text>
-        <Box color='black'>
-          <UnorderedList>
-            {features.map((feature, idx) => (
-              <ListItem key={idx}>{feature}</ListItem>
-            ))}
-          </UnorderedList>
-        </Box>
-      </CardBody>
-      <CardFooter>
-        <Button variant='outline' border='none' color={textColorBrand} _hover={{ textDecoration: 'underline' }}>
-          <Trans i18nKey='pricing_card.view_features'>View all features</Trans>
-        </Button>
-      </CardFooter>
-      {popular && (
-        <Box
-          bgColor='#D2CBB9'
-          w='min-content'
-          whiteSpace='nowrap'
-          mx='auto'
-          py={1}
-          px={3}
-          borderRadius='full'
-          position='absolute'
-          mt={-4}
-          ml='50%'
-          transform='translateX(-50%)'
-          color='black'
-          fontSize='sm'
-        >
-          <Trans i18nKey='pricing_card.most_popular_plan'>Most popular plan</Trans>
-        </Box>
-      )}
-    </Card>
-  )
-}
+}) => (
+  <Card variant='pricing-card' width={width}>
+    <CardHeader>
+      <Text>{title}</Text>
+      <Text>{subtitle}</Text>
+    </CardHeader>
+    <CardBody>
+      <Button variant='pricing-card'>
+        <Trans i18nKey='subscribe'>Subscribe</Trans>
+      </Button>
+      <Text color='black'>
+        <Trans i18nKey='pricing_card.from'>From ${price}/year</Trans>
+      </Text>
+      <Box color='black'>
+        <UnorderedList>
+          {features.map((feature, idx) => (
+            <ListItem key={idx}>{feature}</ListItem>
+          ))}
+        </UnorderedList>
+      </Box>
+    </CardBody>
+    <CardFooter>
+      <Button variant='outline' border='none' _hover={{ textDecoration: 'underline' }}>
+        <Trans i18nKey='pricing_card.view_features'>View all features</Trans>
+      </Button>
```

Comment:
Remove custom styles on buttons. Leave only `variant` prop, everything else will be styled when we have a proper style guide.